nzediegwu1/myblog-api
# #161525705 Setup and configure eslint

#### What does this PR do?
Configure eslint for linting and flagging of code style violations

#### Description of Task to be completed?
- Install eslint and dependencies
- Create .eslintrc file and configure to airbnb style
- Create .gitignore and exclude node_modules
- Create package.json file to specify app settings

#### How should this be manually tested?
1. On your editor, write some codes which violate the airbnb style guide
2. You will observe the violations are flagged
3. Correct the style violations to notice the flags disappear


#### Any background context you want to provide?
Application of airbnb style could require much study of documentation. With eslint setup, linting and flagging of style violations are done automatically, allowing the developer to focus on writing beautiful code

#### What are the relevant pivotal tracker stories?
[#161525705](https://www.pivotaltracker.com/story/show/161525705)

#### Postman Documentation
N/A
